zerver/models.py

```diff
@@ -536,6 +536,8 @@ class UserProfile(ModelReprMixin, AbstractBaseUser, PermissionsMixin):
     """
     EMBEDDED_BOT = 4
 
+    SLASH_COMMANDS = 5
```

Comment:
Would it be possible to add a comment describing how the fields of the user object correspond to the name of the slash command?  (For example, is the name of the slash command stored in the `short_name` field?)